mentorship/gsod/2022/project-ideas.md

```diff
@@ -5,16 +5,84 @@ which Keptn contributors have proposed for this year.
 Application guidelines and more information about Keptn
 in GSoD are available on [this page](../README.md).
 
-## 2022 Project Idea
+## 2022 Project. Re-architecting and improving Keptn user documentation
 
-### Re-architecting and improving Keptn user documentation
+During GSoC 2022 we will run either one large-size project or two small-size ones.
+You will work with the static Keptn website and the documentation engine based on the modern technology stack.
+Currently we use Hugo for the website and Docs, and Google Codelabs for tutorials.
+We plan adoption Docusaurus or Antora as a new documentation platform during Google Summer of Code 2022
+([project idea](https://github.com/keptn/community/blob/main/mentorship/gsoc/2022/project-ideas.md#new-documentation-site-engine)).
+Technical writers in this project will be able to work with one of these technologies while
+using the Documentation-as-Code approach and Mardown/Asccidoc languages.
 
-> Final idea will be published soon.
-> It will be a combination of ideas documented
-> [here](https://docs.google.com/document/d/154zrnkUdUhyWyDB-Gs123JvbiYPb-zQDLaV2Oe0PC8A/edit#heading=h.ckoz6t2d3tab).
+### Key deliverables
+
+We offer the technical writers to focus on 2-3 deliverables from the list below:
```

Comment:
```suggestion
Technical writers should focus on 2-3 deliverables from the list below:
```
I don't know if this is correct but the existing syntax does not work.  Maybe it should be something like "We offer the following list, from which technical writers can choose 2-3 deliverables on which to focus." ?  Or something else.